docs/userguide/mng-storage-driver.md

```diff
@@ -0,0 +1,145 @@
+
+
+## Understand the copy-on-write strategy
+
+Sharing is a good way to optimize resources. People do this instinctively in
+daily life. For example, twins Jane and Joseph taking an Algebra class at
+different times from different teachers can share the same exercise book by
+passing it between each other. Now, suppose Jane gets an assignment to complete
+the homework on page 11 in the book. At that point, Jane can make her own copy
+of page 11, complete the homework, and hand in her copy. The original book is unchanged.
+
+Computer programming developed a similar sharing mechanism for systems called
+the *copy-on-write* (COW) strategy. In this strategy, system processes that use
+the same data share the same instance of that data rather than having their own
+copy. At some point, if one process needs to modify or write to the data, only
+then does the operating system make a copy of the data for that process to use.
+Only the process that needs to write has access to the data copy. All the
+other processes continue to use the original data.
+
+Linux operating systems use the [COW strategy in many
+ways](https://en.wikipedia.org/wiki/Copy-on-write). For example, the  `fork()`
+and `clone()` system calls use a COW strategy when creating processes. Virtual
+memory management and disk storage snapshots also use COW strategies.
+
+## Docker and COW
+
+When you run a Docker container and load an image, Docker uses a copy-on-write
+strategy to optimize the container performance.  For example, consider this
+command line that runs a Python image:
+
+```bash
+$ docker run -ti python bash
+```
+
+If the container changes the `python` image, Docker only tracks (or writes) the
+changes between the container's image and the original image. The original
+`python` image is not modified and any other container also using the `python`
+image are unaffected.
+
+Docker's COW strategy saves space in the container because it need not have a
+whole image locally, only its changes.  And this strategy reduces the time to
+start a container because it downloads only the part of an image it needs to
+run.
+
+Docker's storage backend manages image storage and image change management is
+the **storage driver**. Historically, when Docker first started it relied on the
+AUFS storage driver exclusively to provide this backend service. Now, however,
+Docker supports a number of different storage drivers.
+
+You can choose which storage driver to use when you start the Docker daemon.
+Each storage driver has different strengths and weaknesses. By understanding
+these, you can tune and optimize the performance of your Docker container.
+
+## Understand the storage-driver capabilities
+
+The Docker daemon has support for several different image layer storage
+drivers:
+
+*  AUFS
+*  Device Mapper
+*  BTRFS (ZFS)
+*  Overlayfs
```

Comment:
Yes "VFS" should be listed on its own line and I'm planning on a very short doc for VFS